src/app/item-comments/item-comments.component.html

```diff
@@ -32,20 +33,26 @@
         <span *ngIf="item.type !== 'job'">
         {{item.points}} points by 
           <a [routerLink]="['/user', item.user]" routerLinkActive="active">{{item.user}}</a>
-        </span> 
+        </span>
         <span [class.item-details]="item.type !== 'job'">
           {{item.time_ago}}
           <span *ngIf="item.type !== 'job'"> |
             <a [routerLink]="['/item', item.id]" routerLinkActive="active">
               <span *ngIf="item.comments_count !== 0">
                 {{item.comments_count}}
                 <span *ngIf="item.comments_count === 1">comment</span>
-                <span *ngIf="item.comments_count > 1">comments</span>
-              </span>
-              <span *ngIf="item.comments_count === 0">discuss</span>
-            </a>
-          </span>
-        </span> 
+        <span *ngIf="item.comments_count > 1">comments</span>
+        </span>
+        <span *ngIf="item.comments_count === 0">discuss</span>
+        </a>
+        </span>
+        </span>
+      </div>
+      <div *ngIf="item.type === 'poll'" class="pollResults">
```

Comment:
Right here we can fix this to `*ngIf="item.type === 'poll' && pollResults.length === howManyPollResults"` so all the poll results will render only once they have all loaded completely (take a look at my comment in the `ts` file to see what I mean).

Maybe we can try adding a loading indicator if they all haven't loaded, but don't worry about that for now we can do that later if we want too (and it's unlikely the list of poll results will take a while to load anyway).